source/documentation/using_the_api/api_reference.md

```diff
@@ -101,19 +106,21 @@ Example response:
 }
 ```
 
-### `GET /records/{key}`
-
-Get a specific record within a register based on a particular key. For example, the record for the Borough Council of King's Lynn and West Norfolk in the `local-authority-eng` register. 
+### <a name="get-records-key">Get a specific record within a register based on a particular key</a>
 
-> You must have the exact field value of the unique identifier for the record to get a match from the register. For example, in the `local-authority-eng` register, the field value of the unique identifier (the three-letter code for the county council) must be in capital letters.
+Endpoint: `GET /records/{key}/`
 
-Example URL: `https://local-authority-eng.register.gov.uk/records/KIN`
-
-Example request: `curl --request GET --url https://local-authority-eng.register.gov.uk/records/KIN --header 'Accept: application/json' --header 'Authorization: YOUR-API-KEY-HERE'`
-
-Example response:
+```
+GET /records/{key} HTTP/2
```

Comment:
Here `{key}` should be something specific, given the response, I think the value is `KIN`.